mathics/builtin/colors/color_directives.py

```diff
@@ -660,6 +690,7 @@ class RGBColor(_ColorObject):
     def to_rgba(self):
         return self.components
 
+    rules = {"RGBColor[r, g, b, a]": "{RGBColor[r, g, b], Opacity[a]}"}
```

Comment:
Actually, to make this work as it should in `Background`, we need to improve the support for lists in that option. I think that the best is to remove this rule, and improve the support for the alpha channel in the formatter.

Reply:
Ok. Please put in another PR to make this work better. Thanks.